web-app/package.json

```diff
@@ -40,54 +40,54 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "15.2.10",
-    "@angular/common": "15.2.10",
-    "@angular/compiler": "15.2.10",
-    "@angular/core": "15.2.10",
-    "@angular/forms": "15.2.10",
-    "@angular/platform-browser": "15.2.10",
-    "@angular/platform-browser-dynamic": "15.2.10",
-    "@angular/router": "15.2.10",
-    "@delon/abc": "15.2.1",
-    "@delon/acl": "15.2.1",
-    "@delon/auth": "15.2.1",
-    "@delon/cache": "15.2.1",
-    "@delon/form": "15.2.1",
-    "@delon/mock": "15.2.1",
-    "@delon/theme": "15.2.1",
-    "@delon/util": "15.2.1",
+    "@angular/animations": "^17.3.10",
+    "@angular/common": "^17.3.10",
+    "@angular/compiler": "^17.3.10",
+    "@angular/core": "^17.3.10",
+    "@angular/forms": "^17.3.10",
+    "@angular/platform-browser": "^17.3.10",
+    "@angular/platform-browser-dynamic": "^17.3.10",
+    "@angular/router": "^17.3.10",
+    "@delon/abc": "^17.3.1",
+    "@delon/acl": "^17.3.1",
+    "@delon/auth": "^17.3.1",
+    "@delon/cache": "^17.3.1",
+    "@delon/form": "^17.3.1",
+    "@delon/mock": "^17.3.1",
+    "@delon/theme": "^17.3.1",
+    "@delon/util": "^17.3.1",
```

Comment:
here can we lock the version like before? `"@angular/common": "^17.3.10"`

Reply:
u mean this change cause the error ?

Reply:
> here can we lock the version like before? `"@angular/common": "^17.3.10"`

okay, we will lock the version like before ~

Reply:
> u mean this change cause the error ?

No, just want to lock the version to avoid potential problems. same with yarn.lock